<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>HTML compressor (by aymen mahmoudi) - Version 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup/logo.jpg</normaloff>../../../../.designer/backup/logo.jpg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="entry_html_lineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100000</number>
         </property>
         <property name="placeholderText">
          <string>Paste the original text here</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="out_html_lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>100000</number>
       </property>
       <property name="placeholderText">
        <string>Generated format will appear here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clickHere_pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>this is toolTIP</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(150, 200, 100);
color: rgb(255, 200, 100);</string>
       </property>
       <property name="text">
        <string>Generate compressed format</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
